packages/@okta/vuepress-site/docs/reference/api/asa/attributes/index.md

```diff
@@ -0,0 +1,288 @@
+---
+title: Introduction to Attribute Endpoints
+category: asa
+---
+
+# Attributes API
+
+## Getting Started
+
+This article provides an overview of the attribute API
+
+Explore the Attributes API: [![Run in Postman](https://run.pstmn.io/button.svg)](https://example.com).
+
+
+## Attributes Operations
+
+The Attributes API has the following operations:
+* [List group attributes](#list-group-attributes)
+* [Fetch group attribute](#fetch-group-attribute)
+* [Update group attribute](#update-group-attribute)
+* [Lists the attributes for a user](#lists-the-attributes-for-a-user)
+* [Fetch user attribute](#fetch-user-attribute)
+* [Update a single attribute for a user](#update-a-single-attribute-for-a-user)
+
+
+### List group attributes
+
+<ApiOperation method="GET" url="/v1/teams/{team_name}/groups/{group_name}/attributes" />
```

Comment:
Placeholders in URLs should use the following format: `${nameHere}`. For example `${teamName}`, `${groupName}` and so on.